### PR TITLE
CORE-201 Shut down Community App - Beta,Staging,Test & QA envs

### PR DESCRIPTION
**Pull request by kkartunov**
https://topcoder.atlassian.net/browse/CORE-201
https://topcoder.atlassian.net/browse/CORE-202
https://topcoder.atlassian.net/browse/CORE-250
https://topcoder.atlassian.net/browse/CORE-251

### CODE REVIEW

**Review by jmgasper — APPROVED** [review]
Looks good.  I don't see anything that will break staging, but just wanted to make sure this won't affect staging, since we still use that

**Comment by kkartunov** [comment]
@jmgasper actually we are extending this for staging as well.

**Comment by jmgasper** [comment]
> @jmgasper actually we are extending this for staging as well.

We use staging for QA validation of prod fixes.  We can work around not having it, but it is useful at times.

**Comment by kkartunov** [comment]
@testflyjets FYI.

File: .circleci/config.yml
Comment (testflyjets) [inline .circleci/config.yml:1]: Why not just remove this folder and file entirely? We have version control and can bring it back.

**Review by testflyjets — APPROVED** [review]
Please add a comment at the top of the README saying this code has been DEPRECATED as of today.

File: .circleci/config.yml
Comment (kkartunov) [reply .circleci/config.yml:1]: We need it for prod & dev workflows, those remain.

**Pull request merged**